flow/StreamCipher.cpp

```diff
@@ -19,12 +19,13 @@
  */
 
 #include "flow/StreamCipher.h"
+#include "flow/Arena.h"
 #include "flow/UnitTest.h"
 
 std::unordered_set<EVP_CIPHER_CTX*> StreamCipher::ctxs;
 std::unique_ptr<StreamCipher::Key> StreamCipher::Key::globalKey;
 
-StreamCipher::StreamCipher() : ctx(EVP_CIPHER_CTX_new()) {
+StreamCipher::StreamCipher() : ctx(EVP_CIPHER_CTX_new()), hmacCtx(HMAC_CTX_new()) {
```

Comment:
`hmacCtx` should also get cleaned up in the destructor.

Reply:
Good catch, I missed to free up the context.